webapp/channels/src/components/dropdown_input.tsx

```diff
@@ -143,14 +167,28 @@ const DropdownInput = <T extends ValueType>(props: Props<T>) => {
                         className={classNames('Input', className, {Input__focus: showLegend})}
                         classNamePrefix={'DropDown'}
                         value={value}
-                        onChange={onChange as any} // types are not working correctly for multiselect
+                        onChange={ownOnChange as any} // types are not working correctly for multiselect
                         styles={{...baseStyles, ...styles}}
                         {...otherProps}
                     />
                 </div>
                 {addon}
             </fieldset>
-            {renderError(error)}
+            {error && renderError(error)}
+            {
+                !error && customInputLabel &&
```

Comment:
Why not use `renderError` for the case where a required field is missing? It seems like calling `renderError` with `error || validationError` or something similar would produce the same result

Reply:
Done. Refactored into component to handle both errors together.